tox.ini

```diff
@@ -73,6 +73,7 @@ commands =
 basepython = python3.8
 deps =
     {[testenv]deps}
+    -r docs/requirements.txt
```

Comment:
is this change related to removal of unneeded imports?

Reply:
When cleaning up `pytest.ini`, I accidentally removed the `--ignore=docs/conf.py` line, which made `flake8` tests require doc dependencies. I reverted that line removal, but forgot to revert this. :man_facepalming: